apidoc/Annotation.yml

```diff
@@ -76,6 +76,131 @@ properties:
     type: Titanium.UI.View
     since: "3.1.0"
 
+  - name: showMarker
+    summary: |
+        Boolean to show an annotation view that displays a balloon-shaped 
+        marker at the designated location.
+    description: |
+        Must be set during creation. This is ignored if the customView property is set.
+    type: Boolean
+    default: false
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "11.0"} }
+
+  - name: markerGlyphText
+    summary: The text to display in the marker balloon.
+    description: |
+        Use this property or the markerImage property to specify the marker balloon content. 
+        If you specify both an image and text, the text is displayed.
+        The amount of space available for displaying your glyph text is limited. Specify no more than 
+        two or three characters for any strings you assign to this property.
+    type: String
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "11.0"} }
+
+  - name: markerGlyphColor
+    summary: The color to apply to the glyph text or image.
+    description: |
+        The default value of this property is nil, which applies the standard tint color for the 
```

Comment:
There is no `nil` in Javascript, it's either `null` or `undefined` - I think `undefined` would make more sense here.